lib/modules/hosts/twitchclips.js

```diff
@@ -0,0 +1,16 @@
+export default {
+	moduleID: 'twitchclips',
+	name: 'twitch.tv clips',
+	domains: ['twitch.tv'],
```

Comment:
When #3174 is merged you can change this to `clips.twitch.tv` and remove the hostname check from the line below